code/script.js

```diff
@@ -222,104 +222,154 @@ const generateBoard = () => {
-      )
+      charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value));
+      alert(`Yes the person is a ${value}! Keep all ${value}'s!`);
     } else {
-      alert(
-        `No, the person doesn't wear ${value}! Remove all people that wears ${value}`
-      )
+      charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value));
+      alert(`No the person is not a ${value}, Reamove all ${value}'s!`);
     }
-  } else if (category === 'other') {
-    // Similar to the one above
+  } else if (category === 'accessories') {
+  if (keep) {
+charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value));
+alert(`yes the person wears ${value}! Keep all people with ${value}!`);
+ } else {
+  charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value));
+  alert(`No the person does not wear ${value}! Remove all people with ${value}!`);
+ }
+} else if (category === 'hair') {
+  if (keep) {
+    charactersInPlay = charactersInPlay.filter((person) => person[category] === value);
+    alert(`Yes the person has ${value} ${category}! Keep all people with ${value} ${category}!`);
   } else {
-    if (keep) {
-      // alert popup that says something like: "Yes, the person has yellow hair! Keep all people with yellow hair"
-    } else {
-      // alert popup that says something like: "No, the person doesnt have yellow hair! Remove all people with yellow hair"
-    }
+    charactersInPlay = charactersInPlay.filter((person) => person[category] !== value);
+    alert(`No the person does not have ${value} ${category}! Remove all people with ${value} ${category}!`);
+  }
+} else {
+  if (keep) {
+    charactersInPlay = charactersInPlay.filter((person) => person[category] === value);
+    alert(`Yes the person has ${value} ${category}! Keep all people with ${value} ${category}!`);
+  } else {
+    charactersInPlay = charactersInPlay.filter((person) => person[category] !== value);
+    alert(`No the person does not have ${value} ${category}! Remove all people with ${value} ${category}!`);
   }
-
-  // Determine what is the category
-  // filter by category to keep or remove based on the keep variable.
-  /* 
-    for hair and eyes :
-      charactersInPlay = charactersInPlay.filter((person) => person[attribute] === value)
-      or
-      charactersInPlay = charactersInPlay.filter((person) => person[attribute] !== value)
-
-    for accessories and other
-      charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value))
-      or
-      charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))
-  */
-
-  // Invoke a function to redraw the board with the remaining people.
 }
+  
+  
+ generateBoard();// redraws gameboard
+};
 
-// when clicking guess, the player first have to confirm that they want to make a guess.
+// Function that controlles the users guess
```

Comment:
Function that verifies the user's guess.
Controls is not the same as it would be in Swedish.